redis-lettuce/src/main/java/io/micronaut/configuration/lettuce/cache/RedisCache.java

```diff
@@ -207,6 +215,46 @@ public String getName() {
         return value;
     }
 
+    public <K> Map<K, Object> get(Collection<K> keys) {
```

Comment:
could you add javadoc to all new public methods? Also any return types and method parameters should be annotated with `io.micronaut.core.annotation.Nullable` or `NotNull` to indicate whether the return types or parameters can or cannot be null

Reply:
will do

Reply:
done

Reply:
done